### ISSUE TITLE
[Feature Request] Disable Intellisense suggestions

### ISSUE THREAD

**Issue by ludanin** [ExodiusStudios/vscode-comment-anchors#39]
Hello! Before anything, I'd like to commend how well done and useful this extension is, especially in huge files or big chunks of code.

I've noticed, however, a small annoyance while using Anchors, and that is how IntelliSense suggestions are executed. There's no setting to turn them off, and at least in my usage (mostly go, dart, python & js) any uppercase letter which starts an Anchor (e.g. "R") goes all way to the top in the suggestions list.

Still, I recognize this issue only matters when you're not too lazy to press a single Down on the keyboard. So take your own time to implement this (and possibly there might be a way within vscode to address the order of words that appear inside the IntelliSense).

**Comment by lonix1** [comment]
I agree it would be nice to have a config option to completely disable those snippets / intellisense.

**Comment by disco0** [comment]
In case anyone still wants completion control—I added a completion setting [on my fork](https://github.com/disco0/vscode-comment-anchors):
``` json
"commentAnchors.anchorCompletions": false
```
Atm reloading the window is required to flush the completions (there's a toggle command stub I'll be filling in soon, though)
